packages/syft/src/syft/lib/sympc/rst_share.py

```diff
@@ -0,0 +1,55 @@
+# third party
+from sympc.tensor import ReplicatedSharedTensor
+
+# syft relative
+from ...generate_wrapper import GenerateWrapper
+from ...lib.torch.tensor_util import protobuf_tensor_deserializer
+from ...lib.torch.tensor_util import protobuf_tensor_serializer
+from ...proto.lib.sympc.replicatedshared_tensor_pb2 import (
+    ReplicatedSharedTensor as ReplicatedSharedTensor_PB,
+)
+from .session_util import protobuf_session_deserializer
+from .session_util import protobuf_session_serializer
+
+
+def object2proto(obj: object) -> ReplicatedSharedTensor_PB:
+    share: ReplicatedSharedTensor = obj
+
+    session = protobuf_session_serializer(share.session)
+    proto = ReplicatedSharedTensor_PB(session=session)
+
+    tensor_proto_list = []
+
+    for tensor in share.shares:
+        tensor_proto_list.append(protobuf_tensor_serializer(tensor))
```

Comment:
Q: Why not directly?
`proto.tensor.append(protobuf_tensor_serializer(tensor))`?

Reply:
George ,I think repeated message(TensorData) do not have the append attribute, it is available for the builtin types(string,int32 etc) but not for repeated message attributes. They have add,extend attributes for addition of elements.

Reply:
Then, couldn't we use:
```
proto.tensor.extend(protobuf_tensor_serializer(tensor))
```
?

Reply:
You were right @gmuraru. Its possible, I will make the changes.

Reply:
Yeah @gmuraru the append method works,I think I saw it in a  older documentation of protobuf that append does not work.